src/otx/engine/utils/auto_configurator.py

```diff
@@ -257,3 +268,28 @@ def get_scheduler(self) -> LRSchedulerCallable | None:
         scheduler_config = self.config.get("scheduler", None)
         logger.warning(f"Set Default Scheduler: {scheduler_config}")
         return partial_instantiate_class(init=scheduler_config)
+
+    def get_ov_model(self, model_name: str, meta_info: LabelInfo) -> OVModel:
```

Comment:
How about getting num_classes instead of `meta_info`?

Reply:
To my mind, `meta_info` looks better from the API side. When we call `get_ov_model` we may not bother ourselves with understanding how to obtain num classes from `Label_info`, it'll be done at the lower level. 
Also, current design prevents us from using hacks with explicit `num_classes` modification in the future.

Reply:
I agree with Vlad, communicating with meta_info seems safe and I think it will be extensible in the future, so I'll leave it as is for now.

Reply:
IMHO, giving num_classes seems better. It's because giving `LabelInfo` is actually same as giving availability to use all values in `LabelInfo`, which makes dependency between them. I think the smaller function does the better function is. How do you think about it?

Reply:
I agree your opinion in some degree but honestly I still think it's better to give just num_classes. Dependency can makes it harder to change `LabelInfo` later(I know it barely occurs but as allusion to module), so that's why we try reducing dependency at the most. Anyway, this is about a design preference. So, I'm ok to leave it as it is.